src/components/explorer/GuildCard.tsx

```diff
@@ -33,23 +38,29 @@ const GuildCard = ({ guildData }: Props): JSX.Element => (
   >
     <DisplayCard>
       <SimpleGrid
-        templateColumns={image ? "3rem calc(100% - 4.25rem)" : "1fr"}
+        templateColumns={image ? "3rem calc(100% - 5.25rem)" : "1fr"}
         gap={4}
         alignItems="center"
       >
         {image && <GuildLogo imageUrl={guildData.imageUrl} />}
         <VStack spacing={2} alignItems="start" w="full" maxW="full" mb="0.5" mt="-1">
-          <Text
-            as="span"
-            fontFamily="display"
-            fontSize="lg"
-            fontWeight="bold"
-            letterSpacing="wide"
-            maxW="full"
-            noOfLines={1}
-          >
-            {guildData.name}
-          </Text>
+          <Box position="relative">
+            <Text
+              as="span"
+              fontFamily="display"
+              fontSize="lg"
+              fontWeight="bold"
+              letterSpacing="wide"
+              maxW="full"
+              noOfLines={1}
+            >
+              {guildData.name}
+              {guildData.tags?.includes("VERIFIED") && (
+                <VerifiedIcon iconSize={5} position="absolute" top={1} right={-6} />
```

Comment:
I think if we use an HStack instead of Box as a wrapper, we can remove the relative/absolute positioning and it'd look great without needing to position it this way